app/config.py

```diff
@@ -329,9 +345,12 @@ class Test(Config):
     TRANSIENT_UPLOADED_LETTERS = "test-transient-uploaded-letters"
     LETTER_SANITISE_BUCKET_NAME = "test-letters-sanitise"
 
-    # this is overriden in jenkins and on cloudfoundry
-    SQLALCHEMY_DATABASE_URI = os.getenv(
-        "SQLALCHEMY_DATABASE_URI", "postgresql://postgres:root@localhost:5432/test_emergency_alerts"
+    SQLALCHEMY_DATABASE_URI = "postgresql://{user}:{password}@{host}:{port}/{database}".format(
+        user=os.environ.get("TEST_RDS_USER", "postgres"),
```

Comment:
I'm not sure why the environment variables prefixed with "TEST_" need to be there; why are they different from the others?

Reply:
I'm trying to keep the database used for the unit tests separate from the one used for the actual running code.  This allows them to be the same if you desire, but also allows them to be different.